Source/WarpX.cpp

```diff
@@ -1157,6 +1159,8 @@ WarpX::ReadParameters ()
             maxLevel() == 0 || !do_current_centering,
             "Finite-order centering of currents is not implemented with mesh refinement"
         );
+
+        pp_warpx.query("roundrobin_sfc", roundrobin_sfc);
```

Comment:
This might need a user-facing entry in `parameters.rst`.

Reply:
done